modules/javafx.controls/src/main/java/javafx/scene/control/ContextMenu.java

```diff
@@ -280,6 +309,7 @@ public void show(Node anchor, double screenX, double screenY) {
         if (anchor == null) return;
         if (getItems().size() == 0) return;
         getScene().setNodeOrientation(anchor.getEffectiveNodeOrientation());
+        setAnchorLocation(AnchorLocation.CONTENT_TOP_LEFT);
```

Comment:
Do we need to handle RTL case here?

Reply:
I don't think so.
At least we didn't handle the RTL case here until now. The call to setAnchorLocation(AnchorLocation.CONTENT_TOP_LEFT); is only neccessary to "reset" the anchor location in case we called show with a Side before.